README.md

```diff
@@ -63,9 +63,13 @@ The test-runner [script](scripts/test-runner.sh) is the recommended way for exec
 
 Parameters for the script are controlled by the following environment variables:
 - `TEST_FEATURES`: list of features to be tested.  Subdirectories under `tests` dir that match a feature will be included (internal directories are excluded).  When we have more than one subdirectlory ot tests, they can be listed comma separated.- _required_
-- `NVIDIAGPU_GPU_MACHINESET_INSTANCE_TYPE`: Use only when OCP is on a public cloud, and you need to scale the cluster to add a GPU-enabled compute node. Example instance type: "g4dn.xlarge" in AWS, or "a2-highgpu-1g" in GCP, or "Standard_NC4as_T4_v3" in Azure - _optional_
+- `NVIDIAGPU_GPU_MACHINESET_INSTANCE_TYPE`: Use only when OCP is on a public cloud, and when you need to scale the cluster to add a GPU-enabled compute node. If cluster already has a GPU enabled worker node, this variable should be unset.
```

Comment:
It looks like the example (line 88) needs updating, as it uses `NVIDIAGPU_INSTANCE_TYPE`

Reply:
Good catch !  fixed in next commit.